work_queue/src/work_queue_worker.c

```diff
@@ -2395,6 +2417,7 @@ static void show_help(const char *cmd)
 	printf( " %-30s Set the percent chance per minute that the worker will shut down (simulates worker failures, for testing only).\n", "--volatility=<chance>");
 	printf( " %-30s Set the port used to lookup the worker's TLQ URL (-d and -o options also required).\n", "--tlq=<port>");
 	printf( " %-30s Start an arbitrary process when the worker starts up and kill the process when the worker shuts down.\n", "--coprocess <executable>");
+	printf( " %-30s Specify the number of coprocesses the worker should maintain.\n", "--num_coprocesses=<number>");
```

Comment:
Something like?

Specify the number of coprocesses for serverless functions that the worker should maintain. Default is 0.

We don't want people activating this one thinking it increases the concurrency of all the functions.